src/fsharp/FSharp.Core/prim-types.fs

```diff
@@ -658,7 +658,7 @@ namespace Microsoft.FSharp.Core
                 | _ -> raise (System.IndexOutOfRangeException())
 
             let inline Array2DZeroCreate (n:int) (m:int) = (# "newarr.multi 2 !0" type ('T) n m : 'T[,] #)
-            let GetArray2DSub (src: 'T[,]) src1 src2 len1 len2 =
+            let inline GetArray2DSub (src: 'T[,]) src1 src2 len1 len2 =
```

Comment:
Is there a rationale for why these are now inlined? I don't think it will give any specific performance benefit to inline, and may bloat FSharp.Core ?

Reply:
Having them inline lets us get around a potential compatibility issue. Most of these functions are inlined, and having all of them inline means if the user compiles their code against 5.0 but swaps out the FSharp.Core dll at runtime to 4.7, the compiled code will still work because all these functions have been inlined. Again, this is a super super minor edge case, but this does grant us a bit of extra flexibility here.